ombpdf/document.py

```diff
@@ -143,6 +147,9 @@ def __init__(self, lttextline):
         # very similar left edges are grouped together.
         self.left_edge = Decimal(int((lttextline.x0 - 0.5) / 2) * 2)
 
+    def is_left_edge_near(self, other):
+        return abs(self.left_edge - other.left_edge) <= self.EDGE_THRESHOLD
```

Comment:
Ugggh, for some reason the left edge of the second page of M-15-17 is 4 points to the right of the left edge of page 1... Either that or the metric is inflated b/c of how we're computing left edges in line 148. Anyways, I had to introduce this fuzzy comparison to ensure that the left edge of lines on page 1 are considered as being on the left edge of the page.